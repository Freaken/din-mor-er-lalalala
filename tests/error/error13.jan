// error13: Assertion in from-do-loop-until fails

x -> y;

from x==0 do
  y += x;
  x -= y
loop
  y -= x;
  x += y
until 0<y

