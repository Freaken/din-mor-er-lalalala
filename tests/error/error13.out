Assertion at the beginning of a loop-statement failed
